build.gradle

```diff
@@ -55,14 +55,21 @@ jenkinsPlugin {
 sourceCompatibility = '1.7'
 
 dependencies {
-  compile 'org.jenkins-ci.lib:dry-run-lib:0.1'
-  compileOnly 'org.jenkins-ci:symbol-annotation:1.3'
-  jenkinsPlugins 'org.jenkins-ci.plugins:structs:1.3@jar'
+  jenkinsPlugins 'org.jenkins-ci.plugins:dry-run:0.9'
+  jenkinsPlugins 'org.jenkins-ci.plugins.workflow:workflow-step-api:2.11'
+  jenkinsPlugins 'org.jenkins-ci.plugins:structs:1.7'
+  jenkinsPlugins 'org.jenkins-ci.plugins.workflow:workflow-support:1.14'
 
   signature 'org.codehaus.mojo.signature:java17:1.0@signature'
 
   testCompile 'org.spockframework:spock-core:0.7-groovy-1.8'
-  jenkinsTest 'org.jenkins-ci.main:jenkins-test-harness:2.8@jar'
+  testCompile 'org.jenkins-ci.main:jenkins-test-harness:2.22@jar'
+  testCompile 'org.jenkins-ci.main:jenkins-test-harness-tools:2.2'
+
+  jenkinsTest 'org.jenkins-ci.plugins.workflow:workflow-cps:2.32'
+  jenkinsTest 'org.jenkins-ci.plugins.workflow:workflow-job:2.11'
+  jenkinsTest 'org.jenkins-ci.plugins.workflow:workflow-basic-steps:2.5'
+  jenkinsTest 'org.jenkins-ci.plugins.workflow:workflow-durable-task-step:2.9'
```

Comment:
I see that you depend on `durable-task-step`. Is this used somewhere? I would love the Gradle step to be a durable task step, i.e. when the master loses the connection to the `agent`, the build will continue and the master will re-connect to the agent and resume the operation for where it stopped.

Does this work with the current implementation? Or maybe the step should only support a nice API and the n run an `sh`/`bat` command under the covers.